django/db/models/query.py

```diff
@@ -2462,8 +2477,8 @@ def has_to_attr_attribute(instance):
         attr_found = True
         if rel_obj_descriptor:
             # singly related object, descriptor object has the
-            # get_prefetch_queryset() method.
-            if hasattr(rel_obj_descriptor, "get_prefetch_queryset"):
+            # get_prefetch_querysets() method.
+            if hasattr(rel_obj_descriptor, "get_prefetch_querysets"):
```

Comment:
I'm afraid that to make it fully backward compatible we should check existence of both methods `get_prefetch_querysets` and `get_prefetch_queryset`, e.g.
```python
has_prefetch_qs = (
    hasattr(rel_obj_descriptor, "get_prefetch_querysets")
    or hasattr(rel_obj_descriptor, "get_prefetch_queryset")  # RemovedInDjango60Warning
)
```

Reply:
I changed the condition. It looks worse than on your example but I could not convince black to keep the two conditions on different lines with the comment on the second line.

I assume this should be tested too but I cannot figure out how to to id. Should I create a new field type with a new related manager that do not have the `get_prefetch_querysets` question?